voluncheer/profiles/forms/organizations.py

```diff
@@ -33,13 +37,19 @@ class OrganizationChangeForm(UserChangeForm):
 
     password = None
 
+    photo = forms.ImageField(required=False)
+
     class Meta(UserChangeForm.Meta):
         model = Organization
-        fields = ("name",)
+        fields = ("name", "photo")
 
     def save(self, commit=True):
         user = self.instance
         organization = Organization.objects.get(pk=user)
         if self.is_valid():
             organization.name = self.cleaned_data.get("name")
+            organization.photo = self.cleaned_data.get("photo")
             organization.save()
+        else:
+            logger = logging.getLogger(__name__)
+            logger.exception(self.errors)
```

Comment:
logger.exception is only to be used in an exception handler: https://docs.python.org/3/library/logging.html#logging.Logger.exception

logger.error or logger.warning would be more appropriate.